docs/dev/api_integration.md

```diff
@@ -17,7 +17,7 @@ API integrations send the following types of data to Datadog:
 - [Incidents][7]
 - [Security Events][8]
 
-This page provides instructions for creating an API integration in the `integrations-extras` repository. For more information about why you would want to create an API-based integration, see [Creating your own solution][9]. Since API integrations do not use the Datadog Agent to collect data, you need to create an informational tile-only listing once your development work is complete. 
+This page provides instructions for creating an API integration. REMOVE: For more information about why you would want to create an API-based integration, see [Creating your own solution][9]. Since API integrations do not use the Datadog Agent to collect data, you need to create an informational tile-only listing once your development work is complete. 
```

Comment:
The first two bullets are touched on in this page but added a sentence on write and host their own code: 

> Use a [Datadog API endpoint][1] to enrich and submit data from your backend. API integrations work well in building a connector between Datadog and another SaaS platform. Partners will write and host their code within their enviornment.
This method is ideal for Technology Partners that are SaaS based, and have an existing website for users to log into for authorization purposes.

Reply:
I am unable to find the page that lists all scopes - does this exist?

Reply:
We are already linking to a lot of articles on this page: 
### Create an OAuth client
Instead of requesting API and Application keys directly from a user, Datadog requires using an [OAuth client][14] to handle authorization and access for API-based integrations. For more information, see [OAuth for Integrations][15] and [Authorization Endpoints][16]. 

It may be too much linking to a list of scopes. Is there a way to condense this?

Reply:
I think it would be ideal if all the bullet points I mentioned above were in the overview section. For the OAuth piece, we can leave the scopes link out if it's too much - the oauth docs link to it anyways